### ISSUE TITLE
Re-enable growth plot on the chart "Prirast PO"

### ISSUE THREAD

**Issue by mkadunc** [sledilnik/website#606]
It would be good to show the growth factor (of cases, maybe new hospitalizations and new ICU admissions) - we used to have it on "Prirast PO", but it was computed against cumulative cases and became useless, so we turned it off.

To remedy the issue, we should compute the weekly growth factor using 7d sums today and a week ago. It would be smooth enough and still relatively current information ("only" 1 week old data point). Formula: `growth = cases[0-6]/cases[7-13] - 1 `, this produces percentage growth rate - if it's above 0, it's bad (+66% growth in a week); if it's below 0, it's OK (-20% growth in a week ~ 20% decrease).

**Comment by breki** [comment]
@mkadunc what do I do with this? 
![image](https://user-images.githubusercontent.com/226863/95773132-322c0f80-0cbe-11eb-81cb-a6ad32fbc558.png)

**Comment by breki** [comment]
@mkadunc @lukarenko to be honest, I don't know what I'm supposed to do here. The code for Prirast PO (the 3 disabled pages of that chart) already calculates the growth based on active cases (not cumulative cases). Am I misunderstanding things? Also, the Explainer page (see comment above) uses a fixed 7 days doubling value.

**Comment by breki** [comment]
Preview: https://pr-608.sledilnik.org/sl/stats#spread-chart

**Comment by lukarenko** [comment]
@breki problem is that 3rd chart - Prirast v dnevih:
![image](https://user-images.githubusercontent.com/46861689/95777290-2b08ff80-0cc6-11eb-9abc-3b448cf80165.png)

This is calculated only taking into account new confirmed cases per day, but not taking into account that some will also recover that day. So the growth is faster that it actually is.

**Comment by lukarenko** [comment]
Idea of this chart by @pkese was: to show how small growth contribute to exponential growth, by showing first in days and when the reader has relation of doubles in X days, show what 7 day growth (as example) could look like from current numbers of today

**Comment by breki** [comment]
> @breki problem is that 3rd chart - Prirast v dnevih:
> ![image](https://user-images.githubusercontent.com/46861689/95777290-2b08ff80-0cc6-11eb-9abc-3b448cf80165.png)
> 
> This is calculated only taking into account new confirmed cases per day, but not taking into account that some will also recover that day. So the growth is faster that it actually is.

OK, but how does this relate to @mkadunc description:

> To remedy the issue, we should compute the weekly growth factor using 7d sums today and a week ago. It would be smooth enough and still relatively current information ("only" 1 week old data point). Formula: growth = cases[0-6]/cases[7-13] - 1 , this produces percentage growth rate - if it's above 0, it's bad (+66% growth in a week); if it's below 0, it's OK (-20% growth in a week ~ 20% decrease).

**Comment by breki** [comment]
> Idea of this chart by @pkese was: to show how small growth contribute to exponential growth, by showing first in days and when the reader has relation of doubles in X days, show what 7 day growth (as example) could look like from current numbers of today

Yes, I understand the intention, but I think it would be more useful to show the values for the actual doubling rate, not the example 7-day one.
Also, the number of projected hospitalizations (2528 in 4 weeks) is misleading, since it's way beyond the capacity - maybe we should show the number of people that would not get the place in hospitals instead?

**Comment by mkadunc** [comment]
> Yes, I understand the intention, but I think it would be more useful to show the values for the actual doubling rate, not the example 7-day one.

I agree on this point — making it closer to reality would make it more understandable. We could also do the first panel with "1 week ago", next panel with "now", and then use the same (or similar) factors in subsequent panels...

> Also, the number of projected hospitalizations (2528 in 4 weeks) is misleading, since it's way beyond the capacity - maybe we should show the number of people that would not get the place in hospitals instead?

We could just call it "people requiring hospitalization" - it will also be more accurate if we use admitted to hospital instead of active hospitalized

**Comment by mkadunc** [comment]
> > @breki problem is that 3rd chart - Prirast v dnevih:
> > This is calculated only taking into account new confirmed cases per day, but not taking into account that some will also recover that day. So the growth is faster that it actually is.

Growth in _new cases_  is what we should be looking at - using _active_ only blurs the growth factor and shifts the window more into the past. Same with active hospitalizations and ICUs — both of those are lagging
 
> OK, but how does this relate to @mkadunc description:
> > To remedy the issue, we should compute the weekly growth factor using 7d sums today and a week ago. It would be smooth enough and still relatively current information ("only" 1 week old data point). Formula: growth = cases[0-6]/cases[7-13] - 1 , this produces percentage growth rate - if it's above 0, it's bad (+66% growth in a week); if it's below 0, it's OK (-20% growth in a week ~ 20% decrease).

Related: I think the growth rate plot is wrong - there should be regions when weekly growth rate was negative - e.g. 29.3. to 20.5., 4.7. - 5.8
